entries/p/pandasecurity.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Panda Security": {
+    "domain": "pandasecurity.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.pandasecurity.com/en/support/card?id=700088",
+    "categories": [
+      "security"
+    ]
```

```suggestion
    ],
    "regions": [
      "-cu",
      "-ir",
      "-kp",
      "-ru",
      "-sy"
    ]
```

Per section 19 (Export Restrictions) of [the Legal Notice](https://www.pandasecurity.com/en/homeusers/media/legal-notice/):
> These products may not be downloaded in, nor exported or re-exported to the following countries (or to nationals or residents of these countries): (1) Cuba, Syria, Sudan, North Korea, Iran, the Crimea Region of the Ukraine, (2) any other country subject to U.S. embargoes (3) to anyone on the U.S. Treasury Department's list of Specially Designated Nationals or the U.S. Commerce Department's Table of Denial Orders, (4) or to any person or entity named on the U.S. Department of Commerce's Denied Persons List, Entity List and Unverified List, or the U.S. State Department's List of Statutorily Debarred Parties.

---

Thanks for your checking and additional information! ❤️